qiskit_experiments/library/quantum_volume/qv_analysis.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 import uncertainties
 from qiskit_experiments.exceptions import AnalysisError
-from qiskit_experiments.curve_analysis import plot_scatter, plot_errorbar
+from qiskit_experiments.visualization import plot_scatter, plot_errorbar
```

Comment:
Correct, this analysis class has not been migrated to the new framework yet. The goal is to migrate this to use a plotter, but most likely this will happen in a follow-up PR.